swagger/v1/api_spec.json

```diff
@@ -213,6 +213,205 @@
+                              "email",
+                              "full_name",
+                              "mentor_id",
+                              "school_urn",
+                              "participant_type",
+                              "cohort"
+                            ],
+                            "properties": {
+                              "email": {
+                                "type": "string"
+                              },
+                              "full_name": {
+                                "type": "string"
+                              },
+                              "mentor_id": {
+                                "type": "string"
+                              },
+                              "school_urn": {
+                                "type": "string"
+                              },
+                              "participant_type": {
+                                "type": "string",
+                                "enum": [
+                                  "mentor",
+                                  "ect"
+                                ]
+                              },
+                              "cohort": {
+                                "type": "string"
+                              }
+                            }
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "Unauthorized"
+          }
+        }
+      }
+    },
+    "/api/v1/participants.csv": {
+      "get": {
+        "summary": "Returns all participants",
+        "operationId": "api_v1_participants_index",
+        "tags": [
+          "participant"
+        ],
+        "security": [
+          {
+            "bearerAuth": [
+
+            ]
+          }
+        ],
+        "parameters": [
+          {
+            "name": "filter",
+            "in": "query",
+            "schema": {
+              "type": "object",
+              "example": "",
+              "description": "This schema is used to search within collections to return more specific results.",
+              "properties": {
+                "updated_since": {
+                  "description": "Return participants that have been updated since the date (ISO 8601 date format)",
+                  "type": "string",
+                  "example": "2021-05-13T11:21:55Z"
+                }
+              }
+            },
+            "style": "deepObject",
+            "explode": true,
+            "required": false,
+            "description": "Refine participants to return.",
+            "example": {
+              "updated_since": "2020-11-13T11:21:55Z"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Collection of participants.",
+            "content": {
+              "application/json": {
```

Comment:
This is wrong (in other places also) but I can't figure out how to set it. It still works in the interactive usage

Reply:
It's a bit unfortunate, but I am tempted to say that figuring out why that's happening might be worth more trouble than we want to sink into it at the moment :/ Can we have a tech debt ticket for it?